File: DayTwo/Python/book_suggestion_system.py
import random

list_of_books = ["The Hobbit", "The Mystery", "Animal Farm", "Brave kingdom"]


def get_book_Suggestion():

    return random.choice(list_of_books)
        


def get_page_number():

     return random.randint(1,100)


        
def add_new_books(book_name):

    list_of_books.append(book_name)
    return list_of_books


def remove_books(book_name):

    list_of_books.remove(book_name)
    return list_of_books



def update_books(old_book_name, new_book_name):

    list_of_books.insert(list_of_books.index(old_book_name), new_book_name)
    list_of_books.remove(old_book_name)
    return list_of_books


def show_books():

    return list_of_books
    

    

File: DayTwo/Java/BookSuggestionSystem.java
import java.util.Random;
import java.util.ArrayList;
public class BookSuggestionSystem{

    public ArrayList<String> getListOfBooks(){
    
        ArrayList<String> listOfBooks = new ArrayList<String>();
        listOfBooks.add("The Hobbit");
        listOfBooks.add("The Mystery");
        listOfBooks.add("Animal Farm");
        listOfBooks.add("Brave kingdom");
        
        return listOfBooks;
    }
    
    ArrayList<String> bookList = getListOfBooks();
    
    Random generator = new Random();

    public String getBookSuggestion(){

        int index = generator.nextInt(bookList.size());
        return bookList.get(index);
        
    }

    public int getPageNumber(){

        return generator.nextInt(1,101);

    }
        
    public ArrayList<String> addNewBooks(String bookName){

        bookList.add(bookName);
        return bookList;

    }

    public ArrayList<String> removeBooks(String bookName){
        
        if (bookList.contains(bookName)){
        
            bookList.remove(bookName);
            
        } return bookList;   
    }


    public ArrayList<String> updateBooks(String oldBookName, String newBookName){

        for(int index = 0; index < bookList.size(); index++){
            
            if (bookList.get(index).equals(oldBookName)){
            
                bookList.remove(index);
                bookList.add(index, newBookName);
            
            }
            
        }return bookList;

    }

    public ArrayList<String> showBooks(){

        return bookList;
    
    }

    
}


